cartridges/custom_cartridge_1/cartridge/controllers/Home.js

```diff
@@ -74,14 +74,15 @@ server.extend(module.superModule);
 server.append("Show", function (req, res, next) {
 
     let viewData = res.getViewData();
-   
- 
+
```

Comment:
Do you need all these lines? Lets be consistent

Reply:
I will remove these empty lines.